packages/cli/src/lib/yargs-cli.integration.test.ts

```diff
@@ -30,6 +32,14 @@ describe('yargsCli', () => {
     expect(parsedArgv.onlyPlugins).toEqual([]);
   });
 
+  it('should parse an empty array as a default skipPlugins option', async () => {
```

Comment:
As the `skip` and `only` arguments for plugins are applied after another, I think it would help to document this behaviour in a test.

e.g. 'should use first onlyPlugins and then skipPlugins options on a given config object'

Reply:
great idea!